third-party/rsutils/include/rsutils/type/eth-config-v5.h

```diff
@@ -0,0 +1,45 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2025 RealSense, Inc. All Rights Reserved.
+#pragma once
+
+#include "eth-config-header.h"
+
+
+namespace rsutils {
+namespace type {
+
+
+#pragma pack( push, 1 )
+
+// The structure data for eth config info, version 5
+// See table definition in HWMC v0.54 spec
+// Changes from v4 - 1 reserved byte defined as TTL value.
+struct eth_config_v5
```

Comment:
Please try to use the eth_config_v4, and to see if it can be used for the v5 version.
If all the eth_config_v* can be unified to one unique eth_config.h file, it seems even better to me